app/controllers/users/confirmations_controller.rb

```diff
@@ -6,9 +6,20 @@ def show
     self.resource = resource_class.confirm_by_token(params[:confirmation_token])
 
     if resource.errors.empty?
+      notify_school_primary_contact
       render :confirmed
     else
       respond_with_navigational(resource.errors, status: :unprocessable_entity) { render :new }
     end
   end
+
+private
+
+  def notify_school_primary_contact
+    school = resource.induction_coordinator_profile.schools.first
+
+    if school.primary_contact_email != resource.email
+      UserMailer.primary_contact_notification(resource, school)
+    end
```

Comment:
So we only notify the primary email if the coordinator email is different? Do we not want to send an email to coordinator saying that the process was successful?

Reply:
No coordinator get a message on the ui for now but later they will just get logged in if confirmation is successful

Reply:
When I buy a plane ticket I also see a confirmation, but I also like to get an email 😂 

Just wanna make sure it's something a designer or a product person specified.

Reply:
The flow is user signs up -> user gets "confirm your email" email -> If user confirms email, they get account and we send notification email to school primary contact, right?

Reply:
Yes, @ethanmills . @kicferk1 we are here because we are confirming a registration. Eventually, the user will just get logged in. We can't send them another confirmation that they have confirmed their account.

Reply:
This is not confirming user registration, this is confirming school registration. I think those are quite different in the text they contain, and in the purpose they serve.

Again, I'm happy to not send that email in some scenarios, as long as we confirmed with someone (Abs?) that it's ok to do.

Reply:
This is what was designed and presented to the team. Obviously it's not perfect but for now my job is to implement what was agreed.  I agree the workflow is a bit fuzzy as registering a coordinator automatically leads to registering a school and I'll raise your concerns with  the designer and Abs. If they agree I'll do another pr to change it.

Reply:
Alright, sounds good to me.